packages/components/form/src/form-item.vue

```diff
@@ -50,313 +51,280 @@ import {
   toRefs,
   watch,
   nextTick,
+  useSlots,
 } from 'vue'
-import { NOOP } from '@vue/shared'
 import AsyncValidator from 'async-validator'
+import { clone } from 'lodash-unified'
 import {
   addUnit,
-  isValidComponentSize,
-  getPropByPath,
+  ensureArray,
+  getProp,
+  isString,
+  isBoolean,
 } from '@element-plus/utils'
-import { elFormItemKey, elFormKey } from '@element-plus/tokens'
-import { useSize } from '@element-plus/hooks'
-import LabelWrap from './label-wrap'
+import { formItemContextKey, formContextKey } from '@element-plus/tokens'
```

Comment:
I don't think so, there is nothing benefits by adding `El` as a prefix.

Reply:
I was talking about do we really need the `Context` here? it looks cleaner without it `formItemKey`, since we have marked the type for the key as `InjectionKey<FormItemContext>`

Reply:
Because all our tokens end with `ContextKey`.